WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+final class TitleAndSubtitleTableViewCell: UITableViewCell {
+
+    @IBOutlet var subtitleLabel: UILabel!
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var statusContainerView: UIView!
+    @IBOutlet var statusLabel: PaddedLabel!
+
+    static func register(for tableView: UITableView) {
+        tableView.registerNib(for: self)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureBackground()
+        configureLabels()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+
+}
+
+// MARK: - Setup
+
+private extension TitleAndSubtitleTableViewCell {
+    func configureBackground() {
+        backgroundColor = .listForeground
+        selectedBackgroundView = UIView()
+        selectedBackgroundView?.backgroundColor = .listBackground
+    }
+
+    /// Setup: Labels
+    ///
+    func configureLabels() {
+        subtitleLabel.applyCaption1Style()
+        titleLabel.applyBodyStyle()
+        statusLabel.applyFootnoteStyle()
+        statusLabel.numberOfLines = 0
+        statusLabel.textColor = .black // constant because there will always background color on the label
+        statusLabel.layer.cornerRadius = 2.0
```

Comment:
nit: from testing, the coupon status looks a bit less rounded than the order status. It looks like the order status uses 4.0 for its corner radius, maybe we can stay consistent? or even updating the following `UILabel` extension for generic status label styling?

https://github.com/woocommerce/woocommerce-ios/blob/1010e4a18667062a3fc71850fb339b3b060c32c5/WooCommerce/Classes/Extensions/UILabel+OrderStatus.swift#L51

Reply:
I'm not sure how to update the UILabel extension and keep the changes minimal - so I just simply changed the value in the custom cell: 2a70ab21d77e95d9bc1837685b8f26710adf0ad6.